src/components/IconPopup.scss

```diff
@@ -0,0 +1,37 @@
+$pe-icon-size: 45px;
+
+.pe-icon-popup {
+	background-color: #fff;
+	border-radius: 2px;
+	box-shadow: 0 4px 10px 0 rgba(0, 42, 50, .2);
+	height: $pe-icon-size;
+	padding: 12px;
+	position: absolute;
+	transform: translateX(-50%);
+	width: $pe-icon-size;
+}
+
+.pe-icon-popup::after,
+.pe-icon-popup::before {
+	content: '';
+	height: 0;
+	left: 50%;
+	pointer-events: none;
+	position: absolute;
+	top: -16px;
+	width: 0;
+	z-index: 2;
+}
+
+.pe-icon-popup::after {
+	border: 8px solid transparent;
+	border-bottom-color: #fff;
+	margin-left: -8px;
+}
+
+.pe-icon-popup::before {
```

Comment:
no need to have `::before`. It's for notch border which we don't need.

Reply:
I used notch border is needed to mimic box-shadow